packages/doguAdministration/service.go

```diff
@@ -102,7 +109,7 @@ func (s *server) GetDoguList(ctx context.Context, _ *pb.DoguListRequest) (*pb.Do
 		return nil, err
 	}
 
-	return createDoguListResponse(doguJsonList), nil
+	return s.createDoguListResponse(doguJsonList), nil
```

Comment:
🚫 **[checkstyle]** <contextcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function `createDoguListResponse` should pass the context parameter